tests/observability/metrics/conftest.py

```diff
@@ -1120,3 +1141,17 @@ def windows_vm_for_test(namespace, unprivileged_client):
         storage_class=py_config["default_storage_class"],
     ) as vm:
         yield vm
+
+
+@pytest.fixture(scope="session")
+def memory_metrics_has_bug():
+    return is_jira_open(jira_id="CNV-59679")
```

Comment:
can be used directly in `xfail_if_memory_metric_has_bug`. no need for separate fixture since called only once